src/window.rs

```diff
@@ -284,8 +284,17 @@ impl WindowWrapper {
             (y as f32 / self.renderer.font_height) as u32,
         );
 
-        let sdl_window_wrapper = Sdl2Window::new(&self.window);
-        self.mouse_position = physical_size.to_logical(sdl_window_wrapper.scale_factor());
+        #[cfg(not(target_os = "macos"))]
+        {
+            let sdl_window_wrapper = Sdl2Window::new(&self.window);
+            self.mouse_position = physical_size.to_logical(sdl_window_wrapper.scale_factor());
+        }
+
+        #[cfg(target_os = "macos")]
+        {
+            self.mouse_position = physical_size.to_logical(1.0);
```

Comment:
What happens if the macos device is connected to a higher dpi display?

Reply:
I've switched resolutions on both my monitors, and it seems to work with the change. I'm not sure if my test is exhaustive. This may be a case of "works fine on my machine". I can wait for another Mac user to complain. I'll try to share this around for testing.

Reply:
I think dpi is different from resolution. This stuff has to do with the scaling factor. Unless you explicitly set the dpi, I don't think it will change by just changing the resolution

Reply:
I'll just let this hang while I dig into skulpin, then.